examples/fastspeech2_multispeaker/scripts/train_libri.sh

```diff
@@ -0,0 +1,11 @@
+CUDA_VISIBLE_DEVICES=0 python examples/fastspeech2_multispeaker/train_fastspeech2.py \
```

Comment:
do we need this file ?

Reply:
CUDA preoption?

Reply:
> @machineko i think a command is not clear. It better if you can provide a real command line and also add a structure of file after each step :D. The example dir should rename to fastspeech2_multispeaker_libritts. So they won't mistakenly think that the code is applicable to all dataset / languages :)). Also, maybe add a note for MFA extract in case user want to apply for other languages :D.

Added info in readme